types/three/src/renderers/webxr/WebXR.d.ts

```diff
@@ -26,6 +30,10 @@ export type XRAnimationLoopCallback = (time: number, frame?: XRFrame) => void;
 
 export type XRFrameRequestCallback = (time: number, frame: XRFrame) => void;
 
+export type XRCameraL = PerspectiveCamera & { viewport: Vector4 };
+export type XRCameraR = PerspectiveCamera & { viewport: Vector4 };
+export type XRCamera = Omit<ArrayCamera, 'cameras'> & { cameras: [XRCameraL, XRCameraR] };
```

Comment:
Camera methods come from: https://github.com/mrdoob/three.js/blob/dev/src/renderers/webxr/WebXRManager.js#L51-L63.

Reply:
`WebXR.d.ts` is for covering [WebXR Device API](https://developer.mozilla.org/ja/docs/Web/API/WebXR_Device_API) I think.

I believe this file should not depend on any Three.js components.
You probably have to choose another place to put these type definitions.

Reply:
@joshuaellis any idea? just moving them into `WebXRManager.d.ts` would be good imo.
I would also rename this to `WebXRArrayCamera` .

Reply:
I'm happy for this to be community led, I have never use the WebXR stuff. Will leave with you, @CodyJasonBennett & @0b5vr.

Reply:
https://github.com/three-types/three-ts-types/pull/169/commits/c2535bb6d8acef6534d2d755d00c54df273b7046 : it does look good for me.